frontend/benefit/applicant/src/components/applications/Applications.sc.ts

```diff
@@ -20,6 +20,32 @@ export const $SupplementaryButton = styled(Button)`
   margin-top: ${(props) => props.theme.spacing.xs2};
 `;
 
+export const $Button = styled(Button)`
+  ${(props) =>
+    props.variant === 'primary' &&
+    css`
+      background-color: ${props.theme.colors.coatOfArms} !important;
```

Comment:
is `!important` needed? Or does one of these tricks work? https://styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity

Reply:
fixed